game/views.py

```
@@ -134,8 +134,13 @@ def new_game(request):
 
     if mode not in ('pvp', 'ai'):
         mode = 'pvp'
+    
     player_color = data.get('player_color', 'white')
-    if player_color not in ('white', 'black'):
+
+    if player_color == 'random':
+        import random
+        player_color = random.choice(['white', 'black'])
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)